### ISSUE TITLE
Blacklist of announced addresses

### ISSUE THREAD

**Issue by alexvanin** [nspcc-dev/neofs-node#923]
TLDR: Do not start Storage node if it is configured with localhost in announced addresses.

Storage node has two config section with addresses:
- announced addresses,
- listening addresses.

While listening addresses used  to create a socket on the machine, announced addresses used to connect remote nodes. It seems that it is quite easy to misconfigure them and announce `127.0.0.1`, `localhost` or `0.0.0.0`. Such nodes can be accepted in the network map but they won't be reachable. With enhanced reputation system, such nodes will be dropped from network map and there are no simple way to notify the node owner about misconfiguration.

To solve this issue we can run configuration sanity check before Storage node launch. In fact we already have some sort of sanity checks, but only for attributes. As for announced address, we can restrict any localhost values.

**Comment by alexvanin** [comment]
There was an idea from @cthulhu-rider to implement config sanity checker. User can run sanity check on config file and have some warnings, including the case of local addresses being announced.
